Makefile

```diff
@@ -8,6 +8,8 @@ remove:
 
 start:
 	@docker-compose start
+startanbuild: 
```

Comment:
Typo here?

Reply:
Absolutely, the spelling correction has been made.